lms/envs/devstack.py

```diff
@@ -167,7 +167,9 @@ def should_show_debug_toolbar(_):
     'language': LANGUAGE_MAP,
 }
 
-FEATURES['ENABLE_COURSE_DISCOVERY'] = False
+FEATURES['ENABLE_COURSE_DISCOVERY'] = True
+# Setting for overriding default filtering facets
```

Comment:
I like the comment from common.py better: 

# Setting for overriding default filtering facets for Course discovery